main/ServiceManager/ServiceManager.cpp

```diff
@@ -128,6 +129,10 @@ void ServiceMangerTaskInit()
  */
 void ServiceMangerInit()
 {
+    if (SharedBusInit())
+        ESP_LOGE(TAG, "Failed to Initialize SharedBus.");
+    ESP_LOGI(TAG, "Initialize SharedBus.");
```

Comment:
should be something like: "SharedBus initialized" or "initialized SharedBus successfully"

Reply:
fixed.